package.json

```diff
@@ -31,6 +32,7 @@
     "prepare": "husky"
   },
   "dependencies": {
+    "@aws-sdk/client-apigatewaymanagementapi": "3.699.0",
```

Comment:
planning to contribute to `@effect-aws` to normalize AWS SDK usages. decent stop-gap for now